configs/components/_base-ruby.rb

```diff
@@ -20,15 +20,10 @@
 # SOURCES
 #########
 
-if platform.is_windows?
+if platform.is_windows? && ruby_version_condensed.to_i < 251
```

Comment:
should we just move the `to_i` here to the declaration of `ruby_version_condensed`?

Reply:
No, we use ruby_version_condensed in strings too I believe.

Reply:
Sure, but ruby will auto-stringify a number when it's formatted into a string. I didn't see any use of it in string comparisons

Reply:
Looking at it again: these shouldn't even live in the _base_ruby file anymore. The way we need to manipulate the cmd files to .bat has changed in ruby 2.5, so instead of forcing this branch I just moved these respective branching structures to individual versioned ruby components with no branching.